.github/workflows/pythonapp.yml

```diff
@@ -61,7 +61,7 @@ jobs:
         pytest
     - name: Coverage
       run: |
-        coverage run -a -m pytest tests
+        coverage run -a -m pytest tests --ignore=tests/test_no_extra_install.py
```

Comment:
and this part should no longer be needed due to the skips, although it doesn't hurt.  (maybe you just did a revert on this file?)

Reply:
yep

Reply:
My doubt was whether coverage will get messed up since we generate the coverage when some dependencies are off and then again when they are on, and we run the same tests.